miden/src/repl/mod.rs

```diff
@@ -314,6 +369,8 @@ fn print_instructions() {
     println!("!mem: displays the state of the entire memory");
     println!("!mem[i]: displays the state of the memory at address i");
     println!("!undo: remove the last instruction");
+    println!("!use: display a list of modules available for import");
+    println!("!use <full_module_name>: import provided module to the program");
     println!("!program: display the program");
     println!("!help: prints out all the available commands");
```

Comment:
nit: For `!use <full_module_name>` I would just say "import the specified module".

Also, let's normalize the verbs for all commands - e.g., use "display" instead of "displays", "print" instead of "prints" etc.